AD Force Reset With Slack (#92)
rapid7/insightconnect-workflows: workflows/Force_Password_Reset_in_Active_Directory_With_Slack/help.md

# Description

This workflow triggers from directly slack messaging the chatbot to \"!reset_password\" the defined indicator. This workflow supports automatically Forcing a specified user to reset their password. The workflow will post back results to the specific user.

# Key Features

> * Force a password reset for a specified user

Reviewer: ```suggestion
* Reset an Active Directory user password directly from Slack
```
Keeping consistent wording with the Azure AD help

# Requirements

* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops
* Microsoft Active Directory

# Documentation

## Setup

Once the workflow has been downloaded, login to InsightConnect and "Import" it into the workflow builder.
Once imported, you will initially be prompted to configure the connection for Slack and Active Directory.
In addition The `Find DN` step will need the Search Base field to be defined. This field should be based on your Active Directory domain.

> For example if your domain is example.com then the Search Base would be DC=example,DC=com

Reviewer: ```suggestion
For example if your domain is `example.com` then the Search Base would be `DC=example,DC=com`.
```

# Description

Force an Active Directory user password reset from Slack.

Sample Slack command:

`!reset_password user@example.com`

> 

Reviewer: ```suggestion
```
